Python/mergetwopdf/Readme.MD

```diff
@@ -0,0 +1,23 @@
+# Merge two pdf Files
+
+### Third Party Libraries Required :
+1.PyPDF2
+
+### How to install above Library
+```
+import PyPDF2
+```
+
+### How to use it :
+1. Download or clone the repository
+2. Install Required Libraries
+3. Run merge.py by giving pdf file through command line
+4. you will get the merged pdf in the same directory
+
+### Output
+I have added the merged pdf file named as MergedFiles.pdf which is formed from pdf1.pdf pdf2.pdf .
+You can check the final output in MergedFiles.pdf File

```

Comment:
Can you add a GIF probably showing the output. You can make use of an extension called `Awesome Screenshot` (not an Ad 😆 ) but it is real handy to capture screen and convert into GIF for the purpose.